FiveCalls/FiveCalls/Dashboard.swift

```diff
@@ -60,6 +60,24 @@ struct Dashboard: View {
                 store.dispatch(action: .FetchContacts(location))
             }
         }
+        .onOpenURL(perform: { url in
+            if store.state.issues.isEmpty {
```

Comment:
nit: it'd be great to move this logic to the reducer and middleware

Reply:
I attempted this but it didn't seem to fit with `selectedIssue` being a binding here. Would we move `selectedIssue` into the app state to accomplish this?

Reply:
I think that would be reasonable, while we wouldn't be accessing the selected issue everywhere it is pretty instrumental to the flow of the app.

Reply:
I can't figure out how to structure this right with `selectedIssue` currently on the router, I'm gonna merge and create an issue to take a look again in the future.